Documentation/Contributing/documentation.md

```diff
@@ -36,7 +36,7 @@ When previewing, now you can navigate your browser to [http://127.0.0.1:8000/](h
     Please make sure that your Python binary path is included in your `PATH`.
 
 ## Running helm-docs
```

Comment:
Can we just change this whole section to only talk about running `make docs`? I still think it's nice to keep the developer docs simple. Did you push the changes from when I asked about this previously? Maybe I'm missing the update.

Reply:
I'm suggesting to not even mention `make helm-docs`. Something short like this?
```
## Making docs

[helm-docs](https://github.com/norwoodj/helm-docs) is a tool that generates the documentation for a helm chart automatically. If there are changes in the helm chart, the developer needs to run `make docs` and check in the resulting autogenerated md files at the path `/Documentation/Helm-Charts`. 
```